extract-media

```diff
@@ -12,10 +12,14 @@ _extract() {
   unzip "$TMP_DIR/EXTRACT_MEDIA_TMP.zip" -d "$TMP_DIR/out" > /dev/null
 
   MEDIA_FILES="$(find "$TMP_DIR/out" \( -name "*.png" -o -name "*.jpg" -o -name "*.mp4" -o -name "*.gif" \))"
-
-  echo "$MEDIA_FILES" | while read -r line; do
+  MEDIA_NB=0
+  
+  # echo "$" |
+  while read -r line; do
+    MEDIA_NB=$((MEDIA_NB+1))
     cp "$line" "$2/"
-  done
+  done <<< "$MEDIA_FILES"
+  echo "$MEDIA_NB media succesfully extracted"
```

Comment:
I wonder if there should be a verbose flag or something. Unix tools by default are very quiet. :) Thoughts?

Reply:
This is a good question. Surveying : 
- tar, cp, rm have no output by default, and a verbose option
- wget, zip, git have pretty verbose output and a quiet option. 

Intuitively, modern CLI tend to have some output, probably as they are higher level than base unix commands. 

So I suggest a one liner to give user feedback, and potentially a verbose option with each file and a quiet option

Reply:
Fair enough. :) 

Yeah, I think for now that makes sense but in the long run, I could see a use case for piping the file paths into imageoptim or something. That's for another day then though. :)